phpunit/functional/ConfigTest.php

```diff
@@ -906,7 +909,7 @@ public function testAutoCreateInfocom()
         ];
         $infocom_types = array_diff($infocom_types, $excluded_types);
 
-        $infocom_auto_create_original = $CFG_GLPI["infocom_auto_create"] ?? 0;
+        $infocom_auto_create_original = $CFG_GLPI["auto_create_infocoms"] ?? 0;
```

Comment:
Was the original `infocom_auto_create` key a mistake ?
Because is also used twice later in this function (and 5 times in `ComputerTest`).

If it was a mistake I guess we need to fix all others occurrences.

Reply:
Initially that sounds illogic to take an original value from `infocom_auto_create` and then use it for `auto_create_infocoms`.

Also, `infocom_auto_create` does not seems to exists in GLPI, it's present in tests only. I propose to replace all occurrences with `auto_create_infocoms`.